### ISSUE TITLE
new settings from dashboard

### ISSUE THREAD

**Issue by garanews** [dask/dask-labextension#91]
It would be nice to have the possibility to change directly from dashboard the number of cores and not only the number of workers and also the type of cluster like LocalCluster vs LocalCUDACluster

**Comment by mrocklin** [comment]
I agree.  This would be nice.  Is this something that you're interested in helping with?

**Comment by jsignell** [comment]
Just as a note - this is configurable within the labextension config file as mentioned in the readme. Totally reasonable to also want to expose it in the UI.
